classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.label = self
```

Comment:
This is suppose to create associations between students and classrooms, so it should be `student.classroom = self` instead. :wink: